posthog/api/test/test_external_surveys.py

```diff
@@ -135,7 +135,7 @@ def test_successful_survey_rendering(self):
         self.assertContains(response, "posthog-survey-container")
 
         # Check PostHog configuration is injected
-        self.assertContains(response, "window.projectConfig")
+        self.assertContains(response, "projectConfig")
```

Comment:
my persistent moaning that we get better output from `assert x in y` or `assert x == y` than these methods on self 🙈

Reply:
changed all of them! ty, didn't know about it too